public/app/plugins/datasource/azuremonitor/components/LogsQueryEditor/LogsQueryEditor.tsx

```diff
@@ -87,6 +92,16 @@ const LogsQueryEditor = ({
       onChange(setKustoQuery(updatedBasicLogsQuery, ''));
     }
   }, [basicLogsEnabled, onChange, query, showBasicLogsToggle]);
+
+  useEffect(() => {
+    const timeRange = calculateTimeRange(parseInt(from, 10), parseInt(to, 10));
+    if (showBasicLogsToggle && query.azureLogAnalytics?.basicLogsQuery && timeRange > 8) {
```

Comment:
Nit: might be worth having a comment or casting 8 to a named variable to explain what it is?

Reply:
This has been fixed by @jcolladokuri